docs/shared-content-source/docs/modules/ROOT/pages/app-development-process.adoc

```diff
@@ -250,8 +248,9 @@ Next, we see how to use the application descriptor to deploy the application to
 
 == Deploy with `kubectl` Extensions for a YAML-less Experience 
 
-After testing and packaging, you are ready to deploy the application to a Cloudflow-enabled Kubernetes cluster. 
-In contrast with the usual YAML-full experience that typical K8s deployments require, Cloudflow uses the blueprint information and the streamlet definitions to auto-generate an application deployment. 
+After testing and packaging, you are ready to deploy the application to a Cloudflow-enabled Kubernetes cluster. A Cloudflow-enabled Kubernetes cluster has Cloudflow installed as well as Spark and/or Flink, if your streamlets use them. In contrast with the usual YAML-full experience that typical K8s deployments require, Cloudflow uses the blueprint information and the streamlet definitions to auto-generate an application deployment. 
+
+Kafka is used by Cloudflow to connect streamlets together in a blueprint. If you intend to connect streamlets in this way, at least one Kafka cluster should be available before you install Cloudflow. Cloudflow may be used without Kafka (for example, when your application contains a single streamlet, or an xref:develop:clustering-akka-streamlet.adoc[Akka cluster]), but if your team intends to connect streamlets together and not include Kafka connection information in each topic they define then it's recommended to define a `default` Kafka cluster at install time.
```

Comment:
> together in a blueprint
something like:
"with the topology described in the blueprint"

> If you intend to connect streamlets in this way
well, there is no supported "other way" ATM

Reply:
Important to note that before, the blueprint would connect streamlet with streamlet through inlets and outlets through a connection section. Since 2.0 streamlet inlets and outlets are consumers and producers respectively of topics. And, BTW, a streamlet does not have to have inlets or outlets. It could use some other message queue or database to read from and write to.

Reply:
I added "with the topoloty described in the blueprint" but I'm still not sure how to modify the rest of the para.